### DISCUSSION TITLE
Laravel blade support

### DISCUSSION THREAD

**Discussion by AdiCahyaSaputra** [folke/ts-comments.nvim#49] [Q&A]
How can i create a custom comment string for blade file ?

**Comment by ZayRTun** [comment]
I have been trying to do this too. Any updates?

**Comment by ZayRTun** [comment]
Overriding `html` seems to work, but that is if your working exclusively in blade all the time.
```lua
  {
    'folke/ts-comments.nvim',
    opts = {
      lang = {
        html = '{{-- %s --}}',
      },
    },
    event = 'VeryLazy',
    enabled = vim.fn.has 'nvim-0.10.0' == 1,
  },
```

**Reply by AdiCahyaSaputra** [reply]
Thanks, Im not working with html for a year after I do fullstack web dev with react, and laravel
